File: C++11/백준/Bronze/21638. SMS from MCHS/SMS from MCHS.cc
#include <stdio.h>

int main(){
    int a[2];
    int b[2];
    scanf("%d %d",&a[0],&a[1]);
    scanf("%d %d",&b[0],&b[1]);
    if(b[0]<0 && b[1]>=10){
        printf("A storm warning for tomorrow! Be careful and stay home if possible!");
    }
    else if(b[0]<a[0]){
        printf("MCHS warns! Low temperature is expected tomorrow.");
    }
    else if(b[1]>a[1]){
        printf("MCHS warns! Strong wind is expected tomorrow.");
    }
    else{
        printf("No message");
    }
}

File: Java/백준/Bronze/21638. SMS from MCHS/SMS from MCHS.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int[] a = {in.nextInt(),in.nextInt()};
        int[] b = {in.nextInt(),in.nextInt()};
        if(b[0]<0 && b[1]>=10){System.out.println("A storm warning for tomorrow! Be careful and stay home if possible!");}
        else if(b[0]<a[0]){System.out.println("MCHS warns! Low temperature is expected tomorrow.");}
        else if(b[1]>a[1]){System.out.println("MCHS warns! Strong wind is expected tomorrow.");}
        else{System.out.println("No message");}
    }
}